scripts/shibboleth/README.md

```diff
@@ -0,0 +1,11 @@
+# Shibboleth Configuration for UCL API
+Please note that the following files have been deliberately added to .gitignore for security purposes:

```

Comment:
Can we at all provide examples for some of these?

Reply:
We could do, but there's very little point honestly :/ nobody will be doing our prod deployment. For Dev, use the Dev Dockerfile which I haven't had a chance to test (lol?)